### PR TITLE
Add sequence overflow documentation and example.

### PR DESCRIPTION
**Pull request by andreasscherbaum**

### CODE REVIEW

**Comment by lisakowen** [comment]
just curious @andreasscherbaum - when would one alter the sequence - after the overflow?  or can you alter it at any time after creating it?

**Comment by andreasscherbaum** [comment]
You can alter it at any time. The most likely case is when you have a table where you insert and delete values, and it's only an integer, and this int overflows at some point.

**Comment by mkiyama** [comment]
Couple of comments.

I think the term should be wrapwaround not overflow.

I think you should mention that
-You can set the min and max value for a sequence. I believe the values
 you mentioned are the max for the data type.
-You can specify that the sequence wrap around when you create or alter a sequence
 with the CYCLE keyword.

**Comment by andreasscherbaum** [comment]
@mkiyama I added some of your suggested changes.

For the min value, I have another todo item to add negative steps, and will incorporate the min values there.

File: gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml [hunk 1]
```diff
@@ -92,4 +92,21 @@
...
+          for <codeph>SERIAL</codeph>, <codeph>+9223372036854775807</codeph> for
+          <codeph>BIGSERIAL</codeph>), every <codeph>nextval()</codeph> call will produce an error.
+          Alter the sequence to make it cycle around and start at <codeph>1</codeph> again:</p>
+      <p>
+        <codeblock>ALTER SEQUENCE sequence_name CYCLE;</codeblock>
+      </p>
+      <p>The wraparound behaviour can also specified when a sequence is created manually:</p>
+      <p>
+        <codeblock>CREATE SEQUENCE sequence_name CYCLE;</codeblock>
+      </p>
```
Comment (mkiyama) [inline gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml]: minor edits
line 101 - "call produces an error"

If "sequence_name" in the examples is a variable:
\<varname>sequence_name\</varname>

Comment (andreasscherbaum) [inline gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml]: It's not a variable, but an object name (name of the sequence). Should I still wrap it in <varname>?

Comment (mkiyama) [inline gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml]: OK, this is a minor quibble, but can you change the object name to "read" more like an object and not a variable?
Something like my_new_sequence.

Comment (andreasscherbaum) [inline gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml]: I changed it to "myserial" from the examples above. Better? Or should I rename all of them to "my_new_sequence"?

Comment (mkiyama) [inline gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml]: No, the change looks good to me.
Thanks

**Pull request merged**